relayer/chains/cosmos/sdk_helper.go

```diff
@@ -4,7 +4,6 @@ import (
 	"io"
 	"time"
 
-	"github.com/cosmos/cosmos-sdk/client"
 	cosmosClient "github.com/cosmos/cosmos-sdk/client"
```

Comment:
Since you're touching a line with `cosmosClient` anyway, would you mind renaming `cosmosClient` to `cosmosclient` or just `client`?

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

https://go.dev/blog/package-names

Reply:
Done!